### ISSUE TITLE
Segmentation Fault

### ISSUE THREAD

**Issue by grcasanova** [feimosi/Yet-Another-Pathfinding-Simulator#2]
Using default program, without edits, I get a seg. fault

```
Program received signal SIGSEGV, Segmentation fault.
0x000055555555f27b in yaps::Graph::findPath(yaps::Coordinates, yaps::Coordinates) ()
```

**Comment by feimosi** [comment]
Hi @grcasanova, I'm really sorry but I no longer maintain this project. I'm sure it worked on Linux and Windows 8 back then, so maybe the new versions of some libs break it. I can't help here, but if you find a solution feel free to make a PR.

**Comment by feimosi** [comment]
I've added the `hacktoberfest` label, maybe someone could check the issue on their machine.

**Comment by feimosi** [comment]
Resolved in #3
